RecoJets/JetProducers/plugins/FastjetJetProducer.cc

```diff
@@ -443,7 +443,7 @@ void FastjetJetProducer::runAlgorithm( edm::Event & iEvent, edm::EventSetup cons
       bge_rho->set_particles(fjInputs_);
       fastjet::contrib::ConstituentSubtractor * constituentSubtractor = new fastjet::contrib::ConstituentSubtractor(bge_rho.get());
       // this sets the same background estimator to be used for deltaMass density, rho_m, as for pt density, rho:
-      constituentSubtractor->use_common_bge_for_rho_and_rhom(true); // for massless input particles it does not make any difference (rho_m is always zero)
+      //constituentSubtractor->use_common_bge_for_rho_and_rhom(true); // for massless input particles it does not make any difference (rho_m is always zero)
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant